src/xdocs/config_whitespace.xml

```diff
@@ -430,6 +430,15 @@ class Foo {
 &lt;module name="EmptyLineSeparator"&gt;
   &lt;property name="tokens" value="VARIABLE_DEF, METHOD_DEF"/&gt;
 &lt;/module&gt;
+        </source>
+        <source>
+class FirstClass {
+    int var1 = 1; // ok
+    int var2 = 2; // violation, 'VARIABLE_DEF' should be separated from previous line.
```

Comment:
last update is required.

Please make **absolutely the same java sample code** for all configs. To make it easy for user to see how the same code is going to change is he is changing value of properties.
To archive this, you need to combine few your java codes in one.

I am ok to sacrifice `Example of declarations without empty line separator: ` or `Example of declarations with empty line separator that is expected by the Check by default: `.

Reply:
look at examples of same java code in other Checks - https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/bbc3112_2022223255/config_whitespace.html#NoWhitespaceAfter_Examples

Reply:
Work in progress.

Reply:
Done. Changes have been made.

The result of cli for cofig and java code are below:

```
PS C:\Users\shubh\OneDrive\Desktop\checkstyle project> cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
       "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
        <property name="charset" value="UTF-8" />
  <module name="TreeWalker">
    <module name="EmptyLineSeparator">
  <property name="tokens" value="VARIABLE_DEF, METHOD_DEF"/>
    </module>
  </module>
</module>
PS C:\Users\shubh\OneDrive\Desktop\checkstyle project> cat test.java
///////////////////////////////////////////////////
//HEADER
///////////////////////////////////////////////////

package com.puppycrawl.tools.checkstyle.whitespace;

import java.io.Serializable;

class FirstClass {
  int var1 = 1; // ok
  void method() {} // violation, 'METHOD_DEF' should be separated from previous line.
}
PS C:\Users\shubh\OneDrive\Desktop\checkstyle project> java -jar "C:\Users\shubh\Downloads\checkstyle-10.3.4-all.jar" -c config.xml test.java
Starting audit...
[ERROR] C:\Users\shubh\OneDrive\Desktop\checkstyle project\test.java:11:3: 'METHOD_DEF' should be separated from previous line. [EmptyLineSeparator]
Audit done.
Checkstyle ends with 1 errors.

```

Reply:
Mvn passed without errors

Reply:
Screenshot of the area where changes were made. (I have merged the Example of declarations with empty line separator that is expected by the Check by default and To check empty line before VARIABLE_DEF and METHOD_DEF)

![image](https://user-images.githubusercontent.com/90137881/194827005-7ebee0ea-e763-46ea-aad3-f0be2942fe87.png)